### DISCUSSION TITLE
Pitfalls/Bottlenecks in performance when creating a IO Adapter for Windows.

### DISCUSSION THREAD

**Discussion by izonfreak** [liftoff-sr/CIPster#40] [Q&A] [answered]
I was wondering if there are any possible performance problems if I create and IO Adapter that runs on a normal Windows PC. 
I see a lot of offers/products implementing Ethernet/IP in hardware (e.g. Hilscher) even when the protocol is TCP/IP and UDP/IP and should work out of the box on windows.  I am new to this protocol so I do not have lot of experience with a lot of products, is there something I am missing here ?

**Comment by liftoff-sr** [comment, marked as answer]
On a modern Windows PC you should not have a problem unless you are running multiple higher priority processes than the priority of the process/program that you discuss.  With multiple cores this seems unlikely.
